lib/repositories/file_repository.dart

```diff
@@ -53,4 +66,28 @@ class FileRepository {
     return File.fromJson(result);
   }
 
+  Future<Tuple2<String?, String>> downloadFile({required FileDownloading fileDownloading}) async {
+    var externalStorageDirPath;
+    if (Platform.isAndroid) {
+      externalStorageDirPath =
```

Comment:
Can't we remove  `var externalStorageDirPath; ` and make if final here and below